static/docs/command-reference/checkout.md

```diff
@@ -21,34 +21,31 @@ which instance of each data file or directory should be used, with the checksums
 saved in the `outs` field. The `dvc checkout` command updates the workspace data
 to match with the <abbr>cached</abbr> files that correspond to those checksums.
 
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
+When using Git, the DVC-files can be kept under version control. DVC-files in
+different branches or tags may contain checksums for different data files, saved
```

Comment:
TBH I prefer the previous version ... why do we need to change it?

Reply:
It wasn't great. No need for SCM term, strange phrase "kept under version control" (vs. just "versioned"), "will contain" isn't necessarily true (only a few DVC-files may change among revisions), "command like `git checkout`" instead of just saying "after `git checkout`", confusing phrase "change to the state at the specified branch", "Afterwards" also sounds weird in that place... etc. Most sentences had a small problem so I just rewrote it completely. Maybe I just didn't do a great job. Let me work it some more, trying to recuperate parts of the version you prefer... I'll be back 🕶

Reply:
Ready, can we continue this cat over in https://github.com/iterative/dvc.org/pull/864#pullrequestreview-337940215 below?